sources/Cargo.toml

```diff
@@ -110,6 +110,7 @@ actix-web-actors = { version = "4", default-features = false }
 argh = "0.1"
 async-trait = "0.1"
 aws-config = "1"
+aws-lc-rs = "1"
 aws-sdk-cloudformation = "1"
```

Comment:
Does it make sense to enable bindgen at the workspace level? Do we ever want `lc` without binding generation?

Reply:
I thought so too, but I tried to followed what we do in other crates where we enable the features only when the crates need them. I can go either way, but currently we don't have any crate in `sources/Cargo.toml` that enables features and that's why I chose against this.